### ISSUE TITLE
Language is not changing in React Navigation Drawer?

### ISSUE THREAD

**Issue by cerberusv2px** [stefalda/ReactNativeLocalization#145]
How to change language in the drawer item.
I used the drawer of `react-navigation`. But somehow, other text in the application will change while the text in drawer items will remain as it is. i.e. default language.

**Comment by cerberusv2px** [comment]
Yes. I made a custom drawer and changed the languages there. However, the issues still exists in the toolbar titles and body. So had to change the whole application flow.

**Comment by Biplovkumar** [comment]
same here

**Comment by Biplovkumar** [comment]
please  help for drawer and toolbar titles and body

**Comment by ssghait-aditech** [comment]
@cerberusv2px @Biplovkumar for me only drawer is not getting translated, Any code or document How can ciustomise drawer?

**Comment by estevanpedro** [comment]
can you post your code ? i want to translate all app in the drawer, but i dont know where to put the code to do it in the first place
